frontend2/src/components/tables/chart/EpisodeChart.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useMemo, useState, Fragment } from "react";
+import { Listbox, Transition } from "@headlessui/react";
+import Highcharts from "highcharts";
+import { useEpisodeList } from "api/episode/useEpisode";
+import Icon from "../../elements/Icon";
+import SelectMenu from "../../elements/SelectMenu";
+import HighchartsReact from "highcharts-react-official";
+import TeamChart, { type ChartData } from "./TeamChart";
+import { useSearchTeams } from "api/team/useTeam";
+import { useTopRatingHistoryList } from "api/compete/useCompete";
+import { useQueryClient } from "@tanstack/react-query";
+import * as chart from "./chartUtil";
+import * as random_data from "./randomData";
+
+const EpisodeChart = (): JSX.Element => {
+  const queryClient = useQueryClient();
+  const [episode, setEpisode] = useState("bc23");
```

Comment:
we have an importable helper constant called `DEFAULT_EPISODE` that could be used for a default, or (I'd prefer) could simply query the `name_short` of the currently navigated-to episode id using `useEpisodeId`

Reply:
Also nit: this is an `episodeId` not an `episode` (super minor)